middleware/jwt.js

```diff
@@ -0,0 +1,20 @@
+const passport = require('passport');
+
+const authMiddleware = (req, res, next) => {
+    passport.authenticate(
+        'jwt',
+        {
+            session: false,
+        },
+        (err, user) => {
+            if(!user || err) {
```

Comment:
```suggestion
            if(!user || err || !user.token) {
```
Trzeba jeszcze zobaczyć czy token jest null czy nie